### DISCUSSION TITLE
Chinese text cannot be displayed completely

### DISCUSSION THREAD

**Discussion by JJJoker** [o3de/o3de#11365] [Help Wanted]
I tried to add Chinese text in UI Eitor，but the result was strange. One text display correctly，but another can not display，the text is in the same font.  As shown in the pictures
![image](https://user-images.githubusercontent.com/20180302/185375876-38b2c342-2e77-4017-aefd-a0dd6cc0c5b2.png)
![image](https://user-images.githubusercontent.com/20180302/185376247-2e529c60-2948-4571-8be2-798ff9c61037.png)

**Comment by siliconvoodoo** [comment]
Does it show up if you change 你好 by any other ascii text like "abc"?

**Comment by amzn-rhhong** [comment]
If you change the text from the upper from 你好 to 亚马逊, does it show up? Does it show up if you change it to "你好亚马逊“？
上面那个box里面的字也换成亚马逊的话，还会显示吗？如果换成“你好亚马逊”呢？
这样也许可以排查出是字体本身的问题还是ui系统的问题。

**Comment by slackchen** [comment]
The same to me, I've tested it, it's not font problem,  it's engine that can not shown chinese text correctly.Wish fix it quickly, it looks like a nice engine.

**Reply by yaakuro** [reply]
Read this. It might be a font defintion problem.

https://docs.o3de.org/docs/user-guide/interactivity/user-interface/fonts/rendering/

Adjust the slots number supported by the font definition.

**Reply by slackchen** [reply]
thx~
